magefile.go

```diff
@@ -44,18 +44,20 @@ func (Fmt) Run() error {
 
 // Checks the code formatting.
 func (Fmt) Check() error {
-	goCmd := "gofmt"
-	o, e := sh.OutCmd(goCmd)("-l", ".")
-	if e != nil {
-		return e
+	goModule, err := sh.Output(mg.GoCmd(), "list", "-m")
+	if err != nil {
+		return err
 	}
-	if o != "" {
-		fmtRes, e := sh.OutCmd(goCmd)("-d", ".")
-		if e != nil {
-			return e
-		}
-		return fmt.Errorf("Go code is not formatted:\n\n%s", fmtRes)
+
+	output, err := sh.Output("goimports", "-local="+goModule, "-e", "-d", ".")
+	if err != nil {
+		return err
+	}
+
+	if output != "" {
+		return fmt.Errorf("project source code is not formatted:\n\n%s", output)
```

Comment:
[Same nit](https://github.com/scribd/go-chassis/pull/72#discussion_r983482441) as `go-chassis`.

```suggestion
		return fmt.Errorf("source code is not formatted:\n\n%s", output)
```

Feel free to ignore both.